### DISCUSSION TITLE
TOC 에 h4 태그를 표시해줄 수 있을까요?

### DISCUSSION THREAD

**Discussion by ddaeyeonkim** [pronist/hELLO#80] [Q&A] [answered]
TOC 에 h4 태그를 표시해줄 수 있을까요?

**Comment by pronist** [comment, marked as answer]
공식적으로 ```h4``` 태그의 TOC 표기는 지원할 계획은 없습니다. 일반적으로 **2단**으로만 처리하기 때문입니다. 표시하고자 하신다면 직접 소스코드를 수정하셔서 사용하셔야 합니다. ```script.js``` 에서 ```h.setScrollspy``` 메서드 파라매터에서 ```headings``` 옵션에 ```h4``` 태그를 추가해보시기 바랍니다.

```script.js``` 에서 아래의 부분을 의미합니다.

```js
/* Scrollspy */
$(document).ready(function() {
  // Set
  h.setScrollspy({
    context: '.content__permalink .contents_style',
    spy: '#__spy ul',
    headings: '> h2, > h3'
  });
});
```

**Comment by ddaeyeonkim** [comment]
감사합니다!
